plasma/cli/main.py

```diff
@@ -0,0 +1,13 @@
+import click
+
+
+CONTEXT_SETTINGS = dict(
+    # Support -h as a shortcut for --help
+    help_option_names=['-h', '--help'],
+)
+
+
+@click.group(invoke_without_command=True)
```

Comment:
Needs to be `@click.group(invoke_without_command=True, context_settings=CONTEXT_SETTINGS)` for this to work.